renderer/components/nettests/index.js

```diff
@@ -77,7 +77,8 @@ export const tests = {
   dnscheck: minimalTest('dnscheck'),
   'default': {
     'name': 'Default',
-  }
+  },
+  torsf: minimalTest('torsf')
```

Comment:
nitpick: maybe this item should go before the `default` catch all test_name?

Reply:
The order itself doesn't affect lookups. Although from a readability angle, it makes sense to logically leave the `default` entry at the bottom. I ignored this because I plan to remove all test-specific code for `experimental` anyway.